theme/forms.py

```diff
@@ -275,13 +275,11 @@ def clean_user_type(self):
     def clean_country(self):
         data = self.cleaned_data['country']
         if len(data.strip()) == 0:
-            raise forms.ValidationError("Country is a required field.")
+            raise forms.ValidationError("Country/Region is a required field.")
```

Comment:
Shouldn't this validation error list `Country` only since region is now optional?

Reply:
you're right thanks, fixed